src/pages/home/Home.jsx

```diff
@@ -8,64 +8,117 @@ import FitnessCenterTwoToneIcon from '@mui/icons-material/FitnessCenterTwoTone';
-          direction="column"
-          justifyContent="space-between"
-          spacing={8}
-          my={4}
-        >
-          <Typography variant="h4" color="white">
-            <div>Mysli na sebe i na mateřské dovolené.</div>
-            <div> Buď v kondici. </div>
-            <div> Buď svá. </div>
-          </Typography>
-        </Stack>
-        <Box
+    <>
+      <Box
+        sx={{
+          overflow: 'hidden',
+          height: 600,
+          position: 'relative',
+          '&:after': {
+            content: '""',
+            position: 'absolute',
+            top: 0,
+            bottom: 0,
+            left: 0,
+            right: 0,
+            backgroundColor: 'rgba(0,0,0,0.5)',
+            zIndex: 1,
+          },
+        }}
+      >
+        <CardMedia
           sx={{
-            width: 350,
-            backgroundColor: 'primary',
-            color: 'white',
-            mt: 5,
-
-            p: 1,
+            position: 'absolute',
+            top: 0,
+            bottom: 0,
+            left: 0,
+            right: 0,
+            objectFit: 'cover',
+            objectPosition: 'center 25%',
+            zIndex: 0,
+            height: '100%',
           }}
-        >
-          <p className="motivace">
-            Cvič tam, kde ti to vyhovuje a co tvé tělo potřebuje. Stravuj se
-            tak, aby Ti tvé tělo bylo vděčné a dosáhla jsi svých cílů. Vyplň
-            krátký dotazník a zjisti, jak na to.
-          </p>
-        </Box>
+          image="/assets/home.jpg"
+          title="home"
+          component="img"
+        />
+        <Container sx={{ mb: 10 }}>
+          <Stack
+            direction="column"
+            justifyContent="space-between"
+            spacing={8}
+            sx={{
+              position: 'relative',
+              top: 80,
+              zIndex: 2,
+            }}
+          >
+            <Typography variant="h3" color="white">
+              <div>Mysli na sebe i na mateřské dovolené.</div>
+              <div> Buď v kondici. </div>
+              <div> Buď svá. </div>
+            </Typography>
+          </Stack>
+          <Box
+            sx={{
+              width: 450,
+              color: 'white',
+              p: 2,
+              position: 'relative',
+              top: 155,
+              zIndex: 2,
+              backgroundColor: 'rgba(193,255,200,0.2)',
+            }}
+          >
+            <p className="motivace">
+              Cvič tam, kde ti to vyhovuje a co tvé tělo potřebuje. Stravuj se
+              tak, aby Ti tvé tělo bylo vděčné a dosáhla jsi svých cílů. Vyplň
+              krátký dotazník a zjisti, jak na to.
+            </p>
+          </Box>
 
-        <Stack spacing={2} alignItems="center" my={8}>
-          <Button variant="contained" href={Urls.Questions}>
-            Pojď na to
-          </Button>
-        </Stack>
+          <Stack
+            spacing={2}
+            alignItems="center"
+            my={8}
```

Comment:
remove `margin`, use `bottom` prop to set position of the element.
remove `spacing`, it has no effect with only 1 child element.